app/assets/main/react_components/footprint_result/MembershipAlternative.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { useLocaleData } from '../context/Footprint/LocaleContext.js';
+import PriceTextV2 from './PriceTextV2.jsx';
+import SelectorMultipleTimes from './SelectorMultipleTimes.jsx';
+
+const MembershipAlternative = ({
+  selectedMembership,
+  setSelectedMembership,
+  type,
+  title,
+  sellingPoints,
+  multipleOffsets,
+  setMultipleOffsets,
+  grantedReferralCode,
+  result,
+}) => {
+  const style =
+    'rounded content-between relative sm:pb-8 md:pb-8 lg:w-1/3 m-1 border border-green-accent ' +
+    (type === selectedMembership ? 'bg-green-tint-1 border-2 ' : '');
+
+  for (var point in sellingPoints) {
+    sellingPoints[point] = sellingPoints[point].replace(
+      /\d+/,
+      multipleOffsets.toString()
+    );
+  }
+
+  const {
+    currency: {
+      money: {
+        currency_formats: { [result.plan.price.currency.iso_code]: currency },
+      },
+    },
+  } = useLocaleData();
+
+  const calculatePrice = () => {
+    var price = result.plan.price.subunit_amount / 100;
+    if (Math.trunc(price) != price) {
+      price = price.toFixed(2);
+    }
+    if (type === 'multi') {
+      price = price * multipleOffsets;
+    }
+    if (currency === 'DEFAULT') {
+      price = result.plan.price.currency.iso_code.toUpperCase() + ' ' + price;
+    } else {
+      const findCustomPlacement = /%{.*?}/i;
+      price = currency.replace(findCustomPlacement, price);
+    }
+    return price;
+  };
+
+  return (
+    <div className={style}>
+      <label htmlFor={type}>
+        <div className="h-full m-2 cursor-pointer">
+          <div className="h-3 text-center font-bold">
+            <span>{title}</span>
+          </div>
+          <br></br>
+          <PriceTextV2
+            price={calculatePrice()}
+            grantedReferralCode={grantedReferralCode}
+            selectedMembership={type}
+          />
+          <div className="pl-6">
+            <ul className="list-disc">
+              {Object.keys(sellingPoints).map((point) => {
+                return (
+                  <li className="lg:my-3 text-sm text-left" key={point}>
+                    {sellingPoints[point]}
+                  </li>
+                );
+              })}
```

Comment:
Mer för att visa ett sätt på hur man kan förkorta koden (men du kanske vet detta redan och föredrar return). Obs inte testkört att skriva exakt sådär i koden, så om du vill anamma så pusha inte förslaget direkt utan testkör först =) Och lintern kanske vill ha parenteserna på något annat sätt 
```suggestion
              {Object.keys(sellingPoints).map((point) => (
                  <li className="lg:my-3 text-sm text-left" key={point}>
                    {sellingPoints[point]}
                  </li>
                )
              )}
```